### PR TITLE
Improves the admin room

### PR DESCRIPTION
**Pull request by Daylight2**
## What Does This PR Do

Replaces the meeting room on the admin room with some wizard and syndicate equipment, as well as debug tools.
Replaces the no access requirement airlock on the admin shuttle with a window and a bit of fluff.
Makes the windows of the BSA room indestructible.
Makes uplinks placed down before gamemode is set not cause a runtime.

## Why It's Good For The Game

Players could theoretically get to the admin room on any round in which ERT was spawned. Admins can still teleport trough windows from the right click menu.
Players could either spacewalk to the airlock or abuse z level wrapping to get into the admin room. This fixes both cases, and replaces the very seldom used meeting room with a more practical testing area. 
BSA room windows are indestructible for consistency.
Players can still theoretically float about if they send away the specops shuttle whilst standing in the airlock, but there's nowhere for them to actually go, and I don't want to ruin the aesthetic by surrounding the shuttles with walls on all sides.

## Images of changes

image
image
image

## Testing

Drank the admin booze.
Tested all the debug stuff.
Observed the indestructibility of windows.
Made sure traitor uplinks still worked.

## Changelog
:cl:
fix: Players can no longer raid the admin room.
/:cl:

### CODE REVIEW

**Comment by Daylight2** [comment]
Turns out that you can still hack a CentCom airlock, and I don't think there's a unhackable functional airlock I can use in its place. As recompence for removing their airlock, the admins now get a seat with a view and some admin booze.

**Comment by ArcticValor** [comment]
also wanted to point out that the room with the table in it can be infact teled into via the space loop of the level

**Comment by Daylight2** [comment]
> also wanted to point out that the room with the table in it can be infact teled into via the space loop of the level

Could you elaborate on that? I thought all teleports were disabled on the admin z.

**Comment by ArcticValor** [comment]
> > also wanted to point out that the room with the table in it can be infact teled into via the space loop of the level
> 
> Could you elaborate on that? I thought all teleports were disabled on the admin z.

not tele per say, you know Space Z levels right? basically, if a space Z level does not lead to another it just loops to the other end of the Z level, the admin room that is just on the right-hand side of the part of the map you are editing goes OVER the border that would "tele" the player person to the other side of the map and so with some finagling you COULD (saying you got the positioning right) tele INTO the room with the table from the spec ops shuttle dock and gain access to whatever was in there. (this is very elaborate way of going around this solution but none the less exists)

**Comment by Daylight2** [comment]
> > > also wanted to point out that the room with the table in it can be infact teled into via the space loop of the level
> > 
> > 
> > Could you elaborate on that? I thought all teleports were disabled on the admin z.
> 
> not tele per say, you know Space Z levels right? basically, if a space Z level does not lead to another it just loops to the other end of the Z level, the admin room that is just on the right-hand side of the part of the map you are editing goes OVER the border that would "tele" the player person to the other side of the map and so with some finagling you COULD (saying you got the positioning right) tele INTO the room with the table from the spec ops shuttle dock and gain access to whatever was in there. (this is very elaborate way of going around this solution but none the less exists)

Huh, just tested it, and you can actually teleport into the meeting room on the very east end of the admin shuttle. But I can't really fix that without cutting that entire room off the admin room, so I'll say out of scope (unless the review team thinks that would be a good idea, in which case sure).

**Comment by Daylight2** [comment]
> > > also wanted to point out that the room with the table in it can be infact teled into via the space loop of the level
> > 
> > 
> > Could you elaborate on that? I thought all teleports were disabled on the admin z.
> 
> not tele per say, you know Space Z levels right? basically, if a space Z level does not lead to another it just loops to the other end of the Z level, the admin room that is just on the right-hand side of the part of the map you are editing goes OVER the border that would "tele" the player person to the other side of the map and so with some finagling you COULD (saying you got the positioning right) tele INTO the room with the table from the spec ops shuttle dock and gain access to whatever was in there. (this is very elaborate way of going around this solution but none the less exists)

Alright, I got so annoyed with this that I changed the meeting room. Thanks for the input.

**Comment by Daylight2** [comment]
I can't figure out why the unit tests fail and I need to go for now - would appreciate if someone else took a look.

**Comment by S34NW** [comment]
Looks like you touched some uplinks somewhere in a bad way?

**Comment by Fordoxia** [comment]
Soul removal.

**Comment by TravisAngeI** [comment]
> Soul removal.

What do you mean by this?

I'm assuming that you're meaning people going to the admin shuttle at EoRG but I'm not certain

**Comment by Daylight2** [comment]
> Stealth conflict i believe? it shows as the old sprite in sdmm image

Talked this over on discord, seems to not be an issue.

File: code/game/gamemodes/game_mode.dm [hunk 1]
```diff
@@ -34,7 +34,6 @@
 	var/newscaster_announcements = null
 	var/ert_disabled = FALSE
 	var/uplink_welcome = "Syndicate Uplink Console:"
-	var/uplink_uses = 100
```
Comment (Henri215) [inline code/game/gamemodes/game_mode.dm:37]: You still got unrelated changes

Comment (Daylight2) [reply code/game/gamemodes/game_mode.dm:37]: The uplink uses being declared in gamemode instead of uplink meant that uplinks placed down before roundstart would cause a runtime. The uplink file was the only thing I could find that references it, so I moved it over. This was mentioned in the 'what does this pr do' section.

**Review by Henri215 — APPROVED** [review]
LGTM

**Comment by Fordoxia** [comment]
> > Soul removal.
> 
> What do you mean by this?
> 
> I'm assuming that you're meaning people going to the admin shuttle at EoRG but I'm not certain

Yes

**Review by Contrabang — APPROVED** [review]
No more silly gib guns for the crew, sad. LGTM.
